### DISCUSSION TITLE
Where can I download MACS3 v3.0.2 from?

### DISCUSSION THREAD

**Discussion by kate-wa** [macs3-project/MACS#647] [Q&A] [answered]
Hello,
I had read the documentation on https://macs3-project.github.io/MACS/# which mentions version 3.0.2 that includes new parameters that allow you to customise the cut-of-analysis report.  I hadn't realised that the version available via Pypi is 3.0.1 until I tried to use the new parameters and they weren't available to me.

I have looked around and can't find the version 3.0.2.  Please could you point me in the right direction to get the latest version?

Many thanks, Kate

**Comment by taoliu** [comment, marked as answer]
Hi Kate, we just released the 3.0.2 version to PyPI today. What you saw in June was our unfinished v3.0.2 codes in the `master` branch. Normally we make sure that the codes in the `master` branch pass mandatory checks and tests, so if you are interested in the 'pre-prelease' codes, you can follow this instruction to install through the source code: https://macs3-project.github.io/MACS/docs/INSTALL.html#install-from-source-through-pip
